pkgs/development/tools/rust/cargo-careful/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage {
+  pname = "cargo-careful";
+  version = "master";
+
+  src = fetchFromGitHub {
+    owner = "RalfJung";
+    repo = "cargo-careful";
+    rev = "master";
```

Comment:
The sha256 will break once more commits are pushed to the master branch. Please read https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming and use a specific commit.

Reply:
Turns out the project has releases. I pinned it to the most recent release